geopyspark/tests/png_test.py

```diff
@@ -0,0 +1,37 @@
+import os
+import unittest
+import rasterio
+import numpy as np
+import pytest
+
+from geopyspark.geotrellis.constants import SPATIAL
+from geopyspark.geotrellis.rdd import RasterRDD
+from geopyspark.geotrellis.render import PngRDD
+from geopyspark.tests.base_test_class import BaseTestClass
+
+
+class PngRddTest(BaseTestClass):
+
+    @pytest.fixture(autouse=True)
+    def tearDown(self):
+        yield
+        BaseTestClass.geopysc.pysc._gateway.close()
+
+    def test_if_working(self):
```

Comment:
If there were a more robust system for color mapping, then yes.  We could pull a tile from a tiled rdd, color map it, pack it into an image, and compare it with the result of the PngRDD lookup.  But we don't currently have the facilities for color mapping that would allow this.  I can create a TODO comment in this file to remind us to add a test when we have the ability to do so?